mariza/codepen.html

```diff
@@ -0,0 +1,142 @@
+<header>
+  <div class="header">
+    <center>
```

Comment:
`center` is an outdated HTML element and should not be used: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

Instead, you want to use `text-align: center;` in your CSS.